addons/hr_expense_stripe/views/hr_expense_stripe_credit_card.xml

```diff
@@ -53,8 +50,8 @@
         <field name="model">hr.expense.stripe.credit.card</field>
         <field name="arch" type="xml">
             <list string="Credit Cards" edit="False">
-                <field name="cardholder_id" widget="many2one_avatar_employee"/>
-                <field name="card_number_public"/>
+                <field name="employee_id" widget="many2one_avatar_employee"/>
+                <field name="last_4"/>
```

Comment:
I don't like the "last 4" format, but here it's less problematic than in the card name